node/cloud-functions/README.md

```diff
@@ -0,0 +1,181 @@
+
+When deploying a Cloud Function, make sure you're in the directory that
+contains the `index.js` file you want to deploy:
+
+    cd cloud-functions
+
+Run the following command to deploy the Cloud Function (make sure to replace
+`BUCKET_NAME` with what you used for the bucket name):
+
+    gcloud beta functions deploy helloBigtable --stage-bucket BUCKET_NAME --trigger-topic hello_bigtable
+
+## Triggering the Cloud Function
+
+Run the following command to trigger the Cloud Function:
+
+    gcloud beta pubsub topics publish hello_bigtable '{"prefix": "hello", "count": 10}'
+
+## Verifying everything worked
+
+To see the logs of the Cloud Functions run the following:
+
+    gcloud beta functions logs read --limit 50
+
+You should see something like this:
+
+    D      helloBigtable        78468055256790  2017-11-03 18:35:31.815  Function execution started
+    I      helloBigtable        78468055256790  2017-11-03 18:35:31.904  Received greeting-10
+    I      helloBigtable        78468055256790  2017-11-03 18:35:31.906  starting...
+    I      helloBigtable        78468055256790  2017-11-03 18:35:32.636  10 rows written
+    I      helloBigtable        78468055256790  2017-11-03 18:35:32.703  done!
+    D      helloBigtable        78468055256790  2017-11-03 18:35:32.709  Function execution took 895 ms, finished with status: 'ok'
+
+To see the Bigtable rows run the following:
+
+    cbt read cloud-functions
+
+You should see something like this:
+
+    ----------------------------------------
+    hello-0
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+    ----------------------------------------
+    hello-1
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+    ----------------------------------------
+    hello-2
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "1"
+    ----------------------------------------
+    hello-3
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "1"
+    ----------------------------------------
+    hello-4
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+    ----------------------------------------
+    hello-5
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "1"
+    ----------------------------------------
+    hello-6
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+    ----------------------------------------
+    hello-7
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "1"
+    ----------------------------------------
+    hello-8
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+    ----------------------------------------
+    hello-9
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+
+## Cleaning up
```

Comment:
Do the pub/sub topic and function have to be removed as well?

Reply:
https://cloud.google.com/functions/pricing

It doesn't appear so, the pricing is a function (heh) of invocations and below 2 million has no costs.